movie-browser/src/App.js

```diff
@@ -1,11 +1,7 @@
 import React from 'react';
 
 const App = () => {
-  return (
-    <div>
-      Movie Browser
-    </div>
-  );
-}
+  return <div>Movie Browser</div>;
```

Comment:
Like i said in previous comments. You should always wrapper your return body in parenthesis.